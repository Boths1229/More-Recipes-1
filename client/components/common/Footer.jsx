import React from 'react';

const Footer = () => (
  <footer className="footer-section" id="footer-below">
    <div className="container">
      <strong
        className="copyright"
      >
        More-Recipes Â© 2018, All Rights Reserved, Design &amp; Developed By: SannyTee
      </strong>
    </div>
  </footer>
);

export default Footer;
