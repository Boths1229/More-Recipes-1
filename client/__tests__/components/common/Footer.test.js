import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../../components/common/Footer';

describe('Footer Component', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('strong').length).toBe(1);
    expect(wrapper.find('div').length).toBe(1);
    expect(wrapper.find('footer').length).toBe(1);
  });
});
