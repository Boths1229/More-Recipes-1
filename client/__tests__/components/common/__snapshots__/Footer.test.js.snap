// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should render correctly 1`] = `
<footer
  className="footer-section"
  id="footer-below"
>
  <div
    className="container"
  >
    <strong
      className="copyright"
    >
      More-Recipes Â© 2018, All Rights Reserved, Design & Developed By: SannyTee
    </strong>
  </div>
</footer>
`;
